users/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(write_only=True, min_length=8, required=True)
+
+    class Meta:
+        model = User
+        fields = ['id', 'email', 'password', 'first_name', 'last_name', 'phone']
```

Comment:
The `password` field is defined as `write_only`, but Django automatically includes all fields unless explicitly excluded.
Add `extra_kwargs` to ensure password is not accidentally exposed in responses.
```
class Meta:
    model = User
    fields = ['id', 'email', 'password', 'first_name', 'last_name', 'phone']
    extra_kwargs = {
        "password": {"write_only": True},
    }
```